docs/install-upgrade/build-wiring.md

```diff
@@ -3,36 +3,74 @@
-   --fabric-links-count value   number of fabric links if fabric mode is spine-leaf (default: 0)
-   --mclag-leafs-count value    number of mclag leafs (should be even) (default: 0)
-   --mclag-peer-links value     number of mclag peer links for each mclag leaf (default: 0)
-   --mclag-session-links value  number of mclag session links for each mclag leaf (default: 0)
-   --orphan-leafs-count value   number of orphan leafs (default: 0)
-   --spines-count value         number of spines if fabric mode is spine-leaf (default: 0)
-   --vpc-loopbacks value        number of vpc loopbacks for each switch (default: 0)
+   --help, -h  show help
 ```
+
+## Design Discussion
+This section is meant to help the reader understand how to assemble the primitives presented by the Fabric API into a functional fabric. This discussion starts with the primary building block of your design, a VPC.
+
+### VPC
+
+A VPC allows for isolation at layer 3. This is the main building block for users when creating their architecture. Hosts inside of a VPC will see each other but nothing else. The hosts inside of a VPC will likely need to connect to other VPCs or the outside world. To communicate between two VPC a *peering* will need to be created. A VPC can be a logical separation of workloads. By separating these workloads additional controls are available. The logical separation doesn't have to be the traditional database, web, and compute layers it could be development teams who need isolation, it could tenants inside of an office building, or any separation that allows for better control of the network. Once your VPCs are decided, the rest of the fabric will come together. With the VPCs decided traffic can be prioritized, security can be put into place, and the wiring can begin. The fabric allows for the VPC to exist beyond a single switch, which gives flexibility when the physical world meets the digital.
+
+### Connection
+
+A connection represents the physical wires in your data center. They connect switches to other switches or switches to servers.
+
+#### Server Connections
+
+A server connection will require server side configuration as the Fabric configuration abilities do not reach into the end hosts. A server connection can be one of: 
+
+- *Unbundled* - a single cable going from switch to server
+- *Bundled* - two or more cables going to a single switch, the server needs to configured for this, Fabric handles the switch
+- *MCLAG* -  two cables going to two different switches, also called dual homing. The switches will need a fabric link between them
+- *ESLAG* - two to four cables going to different switches, also called multi-homing. If four links are used there will be four switches connected to a single server with four NIC ports
+
+#### Fabric Connections
+
+These serve as connection between switches, their beautiful weave comprises the fabric of the network.
```

Comment:
```suggestion
Fabric connections serve as connections between switches, they form the fabric of the network.
```

Can be confusing. I know, I'm a party-breaker :grimacing:

Reply:
poetry :)